### ISSUE TITLE
Bankruptcy Transaction Type

### ISSUE THREAD

**Issue by alfbrown** [eprbell/dali-rp2#272]
I am working on building a simple GUI for the project and ran into an issue during testing and wanted to confirm that Bankruptcy (Recovery/Liquidation) are unsupported Transaction types?

**Comment by eprbell** [comment]
Thanks for working on a GUI! Regarding your question, I'm not sure: can that be captured with existing transaction types? And if not, what cannot be expressed, if you use existing types? BTW, from similar discussions in the past, I think we'll need a new type for lost/stolen coins.

**Comment by alfbrown** [comment]
![screencapture-localhost-8080-2025-01-11-02_32_57](https://github.com/user-attachments/assets/509357a1-50c8-4aad-b655-d95a5efad359)
I've been playing around with it, and I think the existing sites can cover this issue. BTW, attached is a screenshot of the site (a few fixes still need to be made).

**Comment by eprbell** [comment]
Looks interesting! is it free or commercial, open-source or closed-source? Also is it privacy-focused? Thanks for the attribution.

**Comment by alfbrown** [comment]
Yes, it’s free, I may add ads to cover potential cost but as for now, not sure how much traffic this will get. The backend is built on DALI and RP2 so I planned to point users to that. Also, I have been considering and would open-source the modified backend but haven’t made a decision. Yes, privacy is first. The backend will not maintain any user files for. I envision this is just the frontend UI to this project, which is why I am aiming for this to still be free.
